### PR TITLE
mongodb buster upgrade

### PR DESCRIPTION
**Pull request by alexandrasp**
update mongodb to 4.2 using official repos in buster.

Signed-off-by: Alexandra Pereira <alexandra.pereira@collabora.com>

### CODE REVIEW

File: roles/install-deps/tasks/install-mongodb.yml [hunk 1]
```diff
@@ -86,8 +86,8 @@
     - mongodb
     - deps
 
-- name: Install MongoDB
-  when: ansible_distribution == "Debian" and ansible_distribution_major_version == "8"
+- name: Install MongoDB on Stretch
```
Comment (gctucker) [inline roles/install-deps/tasks/install-mongodb.yml]: I thought we were on Buster i.e. Debian 10?

Comment (alexandrasp) [reply roles/install-deps/tasks/install-mongodb.yml]: yes, we were. We already have entry to install mongo db on [Buster,](https://github.com/kernelci/kernelci-backend-config/blob/master/roles/install-deps/tasks/install-mongodb.yml#L61) it was only necessary update the repo and let this entry to install stretch.

Comment (gctucker) [reply roles/install-deps/tasks/install-mongodb.yml]: OK, but then it's the default package from Debian Stretch which I'm not sure will actually work.

That's fine for now as the initial requirement is to have it working with Mongo v4.2 on Buster, if it works or not on other distros isn't too important.  But as a follow-up, we should decide which distro version we actually support and either test them or drop them.  In doubt, we can just drop everything except Buster.

@khilman Which distro are you using in the Docker image for kernelci-backend?

Comment (alexandrasp) [reply roles/install-deps/tasks/install-mongodb.yml]: ok, I will update to have only buster as option.

Comment (alexandrasp) [reply roles/install-deps/tasks/install-mongodb.yml]: @gctucker I removed all that are not related to buster, let me know if I missed anything.

**Comment by gctucker** [comment]
Thanks, would be good to test that in a Buster Docker image among other things.

@khilman Is this something you want to do?

**Comment by khilman** [comment]
I've tested a basic buster + mongodb 4.2 upgrade of kernelci-docker here: https://github.com/BayLibre/kernelci-docker/tree/dev/upgrade-buster

It's doing a similar change for the repo, so LGTM.

**Comment by gctucker** [comment]
I've tested that on a Debian Buster VM image from scratch to be able to redeploy the staging server.  Just had one glitch with the `mongod` service name, see #27.

Also there seems to be support for some AWS backups but I don't think we have an active account for that any more, so the `backup` tagged tasks should probably be optional.

**Pull request merged**